templates/typescript-alloy-app/.gitignore

```diff
@@ -0,0 +1,16 @@
+.DS_Store
+/Resources
+/platform
+/i18n
+build.log
+build
+npm-debug.log
+tmp
+.map
+.project
+.settings
+Thumbs.db
+plugins/ti.alloy
```

Comment:
We should not put that on .gitignore as it's causing a build failure when missing this file. It should reaaaally not love in the every project anyway, but until thats fixed, let's restore the dir and remove it from here (ryming like boss).